src/main.rs

```diff
@@ -118,7 +131,20 @@ fn settings() -> io::Result<Settings> {
         },
     )?;
 
-    Ok(Settings { threshold })
+    let keep = match matches.values_of(KEEP_OPTION) {
+        Some(values) => match RegexSet::new(values) {
+            Ok(set) => Some(set),
+            Err(_) => {
+                return Err(io::Error::new(
+                    io::ErrorKind::InvalidInput,
+                    "Invalid regex format provided",
```

Comment:
How ugly would it be if we just used the error message from `RegexSet::new`? I think it would be good to include the original error message so the user knows what they messed up, but not if it's too ugly.

Reply:
I made this change. The format isn't very ugly and it is way more helpful. Example:

```
[2021-07-13 21:17:11 -04:00 ERROR] regex parse error:
    (\bkong\b|
    ^
error: unclosed group
```

Reply:
Hah, I think it's kind of ugly since the message isn't capitalized and isn't a sentence, unlike all the error messages we emit. But I agree it's more helpful!